gcalcli/gcalcli.py

```diff
@@ -915,14 +918,15 @@ def _SetEventStartEnd(self, start, end, event):
         event['s'] = parse(start)
         event['e'] - parse(end)
 
-        if self.options['allday']:
+        try:
+            self.options['allday']
```

Comment:
While I appreciate the pythonic style of EAFTP, I don't really like what's happening on 922.  It seems like this is only here to trigger the KeyError (correct me if I'm wrong).  If the only point of the KeyError is to handle the case where we don't have 'allday' then I think I'd prefer to use `.get` for this.

if self.options.get('allday'):  # returns None if there is no such key.

Reply:
Yep, ok. Agree with this.